doc/nrf/gs_ins_windows.rst

```diff
@@ -273,8 +316,41 @@ To install those, open a |bash| in the ``ncs`` folder and enter the following co
 Setting up the build environment
 ********************************
 
+Before you start :ref:`building and programming a sample application <gs_programming>`, you must set up your build environment.
+
+Setting up the SES environment
+==============================
+
+If you plan to :ref:`build with SEGGER Embedded Studio <gs_programming>`, the first time you import an |NCS| project, SES will prompt you to set the paths to the Zephyr Base directory and the GNU ARM Embedded Toolchain.
```

Comment:
It would be better to add an ID for "Building with SES" in gs_programming.rst and link to that ID instead of to the top of that page. If we add more options for building later, the link will go to the right place.